src/Storage/Storage.Management/Az.Storage.psd1

```diff
@@ -57,7 +57,17 @@ RequiredModules = @(@{ModuleName = 'Az.Accounts'; ModuleVersion = '1.3.0'; })
 
 # Assemblies that must be loaded prior to importing this module
 RequiredAssemblies = '.\Microsoft.Azure.Management.Storage.dll', 
-               '.\Microsoft.WindowsAzure.Storage.dll'
+               '.\Microsoft.Azure.Storage.Common.dll',
+               '.\Microsoft.Azure.Storage.Blob.dll',
+               '.\Microsoft.Azure.Storage.File.dll',
+               '.\Microsoft.Azure.Storage.Queue.dll',
+               '.\Microsoft.Azure.Cosmos.Table.dll',
+               '.\Microsoft.Azure.DocumentDB.Core.dll',
```

Comment:
It looks like there are dependency conflicts with other modules (System.Reflection.TypeExtensions) that will need to be addressed.

Reply:
Why ios there a dependency on DocumentDB?

Reply:
`Microsoft.Azure.DocumentDB.Core.dll` is referenced by new table lib `Microsoft.Azure.Cosmos.Table.dll`

Reply:
For "System.Reflection.TypeExtensions", what's impact of the conflict? Build failure? Runtime Failure? Test Failure? How to address it?